opensrp-tasking/src/main/java/org/smartregister/tasking/util/GeoJsonUtils.java

```diff
@@ -0,0 +1,190 @@
+                interventionList.append("~");
+
+            }
+
+            populateBusinessStatus(taskProperties, mdaStatusMap, state);
+
+            taskProperties.put(TaskingConstants.Properties.TASK_CODE_LIST, interventionList.toString());
+            if (structureNames.get(structure.getId()) != null) {
+                taskProperties.put(Constants.DatabaseKeys.STRUCTURE_NAME, structureNames.get(structure.getId()).getStructureName());
+                taskProperties.put(TaskingConstants.Properties.FAMILY_MEMBER_NAMES, structureNames.get(structure.getId()).getFamilyMembersNames());
+            }
+            structure.getProperties().setCustomProperties(taskProperties);
+
+        }
+        return BaseInteractor.gson.toJson(structures);
+    }
+
+    public void calculateState(Task task, StateWrapper state, @NonNull Map<String, Integer> mdaStatusMap) {
+//        if (Utils.isResidentialStructure(task.getCode())) {
+//            switch (task.getCode()) {
+//                case TaskingConstants.Intervention.REGISTER_FAMILY:
+//                    state.familyRegTaskExists = true;
+//                    state.familyRegistered = COMPLETE.equals(task.getBusinessStatus());
+//                    state.ineligibleForFamReg = NOT_ELIGIBLE.equals((task.getBusinessStatus()));
+//                    break;
+//                case BEDNET_DISTRIBUTION:
+//                    state.bednetDistributed = COMPLETE.equals(task.getBusinessStatus()) || NOT_ELIGIBLE.equals(task.getBusinessStatus());
+//                    break;
+//                case BLOOD_SCREENING:
+//                    if (!state.bloodScreeningDone) {
+//                        state.bloodScreeningDone = COMPLETE.equals(task.getBusinessStatus()) || NOT_ELIGIBLE.equals(task.getBusinessStatus());
+//                    }
+//                    state.bloodScreeningExists = true;
+//                    break;
+//                case CASE_CONFIRMATION:
+//                    state.caseConfirmed = COMPLETE.equals(task.getBusinessStatus());
+//                    break;
+//                case MDA_ADHERENCE:
+//                    state.mdaAdhered = COMPLETE.equals(task.getBusinessStatus()) || NOT_ELIGIBLE.equals(task.getBusinessStatus());
+//                    break;
+//                case MDA_DISPENSE:
+//                    populateMDAStatus(task, mdaStatusMap);
+//                    break;
+//                default:
+//                    break;
+//            }
+//        }
+    }
+
+
+    public void populateBusinessStatus(HashMap<String, String> taskProperties, Map<String, Integer> mdaStatusMap, StateWrapper state) {
+        // The assumption is that a register structure task always exists if the structure has
+        // atleast one bednet distribution or blood screening task
+//        if (Utils.isResidentialStructure(taskProperties.get(TASK_CODE))) {
+//
+//            boolean familyRegTaskMissingOrFamilyRegComplete = state.familyRegistered || !state.familyRegTaskExists;
```

Comment:
I am not sure of the intention when commenting these out. Could you do something about it? Create an issue, todo, remove, fix